spec/runner_spec.rb

```diff
@@ -484,6 +484,10 @@
       expect(run('test', '--help')).to eq("Implement help for test here\n")
     end
 
+    it 'can be used after the command and command arguments' do
+      expect(run('test', 'command-arg', '--help')).to eq("Implement help for test here\n")
+    end
```

Comment:
Let's also add a similar test for the `help test command-arg` scenario

Reply:
And just to be thorough, a test for multi-word commands would be nice — something like the following:

```ruby
it 'can be used before a multi-word command with command arguments' do
  expect(
    run('help', 'module', 'install', 'command-arg') do
      command('module install') { |c| c.when_called { say 'whee!' } }
    end
  ).to eq("Implement help for module install here\n")
end
```

Reply:
Added in [ea4d912](https://github.com/commander-rb/commander/compare/b6381fc67faef3872c44e0e0f19fd030d6a67ea6..ea4d912a58ec5332fbd605a1e0d91bf3230f4160).